.gitlab-ci.yml

```diff
@@ -57,7 +57,7 @@ gcp-setup:
   script:
     - docker version
     - docker-compose version
-    - docker pull gpii/exekube:0.4.0-google
```

Comment:
Is this temporary till we get proper versioning for exekube?

Reply:
This is updating gcp-setup from an older approach to the current state of the art. Updating Exekube versioning is in https://issues.gpii.net/browse/GPII-3745.